packages/xrpl/HISTORY.md

```diff
@@ -2,7 +2,13 @@
 
 Subscribe to [the **xrpl-announce** mailing list](https://groups.google.com/g/xrpl-announce) for release announcements. We recommend that xrpl.js (ripple-lib) users stay up-to-date with the latest stable release.
 
-## Unreleased Changes
+## Unreleased
+
+### Added
+* `convertTxFlagsToNumber` as a utility function in xrpl to return the number conversion of a transaction's flags
```

Comment:
Not necessary to explain the the function since the name implies its functionality. One can look at the JSDoc for explanation.

```suggestion
* Adds utility function `convertTxFlagsToNumber`
```